cmake/toolchain-mingw-i686.cmake

```diff
@@ -1,13 +1,16 @@
-SET(CMAKE_SYSTEM_NAME Windows)
+set(CMAKE_SYSTEM_NAME Linux)
```

Comment:
This is wrong, MinGW is not Linux...

Reply:
Fixed.

Reply:
Basically what this line does is for example that it changes the resource file format to PE instead of ELF and use windres to embed them to executable.